packages/amazonq/.changes/next-release/Bug Fix-c2494a35-1254-4920-abec-2fb44a15ac1d.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Fix a bug where subsequent /review did not work after reviewing once"
```

```suggestion
	"description": "/review: subsequent reviews weren't possible"
```
or something like that